addons/pos_restaurant/models/pos_restaurant.py

```diff
@@ -78,5 +78,7 @@ class RestaurantPrinter(models.Model):
     _description = 'Restaurant Printer'
 
     name = fields.Char('Printer Name', required=True, default='Printer', help='An internal identification of the printer')
+    printer_type = fields.Selection(string='Printer Type', default='iot',
+        selection=[('iot', ' Use a printer connected to the IoT Box')])
```

Comment:
you can probably put ('iot', ' Use a printer connected to the IoT Box') in module pos_restaurant_iot. check with laura if we want to keep in 13.0 the ability to use the iot box in community for printers.